src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/ManifestMetadata.cs

```diff
@@ -187,7 +187,7 @@ public Uri ProjectUrl
 
         public RepositoryMetadata Repository { get; set; }
 
-        private IEnumerable<PackageDependencyGroup> _dependencyGroups = new List<PackageDependencyGroup>();
+        private IEnumerable<PackageDependencyGroup> _dependencyGroups = [];
```

Comment:
nit: we could make the intent clear by
```suggestion
        private IEnumerable<PackageDependencyGroup> _dependencyGroups = Enumerable.Empty<PackageDependencyGroup>();
```

Reply:
Under the hood, they're the same thing in .NET Core.

Reply:
@Nigusu-Allehu there are a significant number of places in this code base where the intent is clear, but is results in sub-optimal code. the `[]` under the hood is smart enough to chose the best type for the majority of scenarios. IMO we should let it choose.

but it is your code base so happy to be overruled

Reply:
Either is fine with me as well. We can mark this as resolved